src/seedu/addressbook/AddressBook.java

```diff
@@ -1144,14 +1152,14 @@ private static String getUsageInfoForExitCommand() {
      */
 
     /**
-     * Removes sign(p/, d/, etc) from parameter string
+     * Removes prefix(p/, d/, etc) from parameter string
      *
-     * @param s  Parameter as a string
-     * @param sign  Parameter sign to be removed
-     * @return  string without the sign
+     * @param fullString  Parameter as a string
+     * @param prefix  Parameter prefix to be removed
+     * @return  string without the prefix
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
```

Comment:
the solution to this one is incorrect.
this one is pretty much the same as earlier. you need to change the code so that only prefix is removed.
in the current version all occurrences of `prefix` in `fullString` are removed.